src/guide/typescript/composition-api.md

```diff
@@ -4,6 +4,8 @@
 
 ## Typing Component Props
 
+### using `<script setup>`
```

Comment:
@jCodeLife that's a nice addition! Could we please capitalize the first letter to be consistent with other headings? 🙏🏻 
```suggestion
### Using `<script setup>`
```

Reply:
Yes,you are right